util/cron/test-linux64.incr.bash

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+#
+# Test incremental compilation on examples only,
+# using default linux64 configuration.
+
+CWD=$(cd $(dirname $0) ; pwd)
+source $CWD/common.bash
+
+export CHPL_NIGHTLY_TEST_CONFIG_NAME="linux64.incr"
+
+$CWD/nightly -cron -examples --incremental
```

Comment:
I suspect this line needs to read:

$CWD/nightly -cron -examples -compopts --incremental

instead, as I don't think nightly will just detect that flag automatically.